janusgraph-cassandra/src/test/resources/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
Drop line?

Reply:
@mbrukman I did remove this line. Not sure why the comment isn't showing as outdated but I think I've seen that behavior before when the comment was on the first line in the file.

Reply:
Agreed, this is weird. The comment is misaligned given the new changes.